### ISSUE TITLE
ErrorException in LdapAuthUserProvider.php line 257: Undefined offset: 1

### ISSUE THREAD

**Issue by elite123** [strebl/ldap-auth#14]
I'm getting this after calling \Auth::validate(), when I have 'recursive_groups' => true,

```
if (\Auth::validate(['username' => $input['username'], 'password' => $input['password']])) {
    // Authentication passed...
    return redirect()->intended('home');
}
```

**Comment by strebl** [comment]
Hmm line [257](https://github.com/strebl/ldap-auth/blob/v2.1.2/src/LdapAuthUserProvider.php#L257) is a comment.

Which package version do you use?

**Comment by strebl** [comment]
I think #15 is the same issue but for the most recent version.

I'll close this issue and fix #15.

Feel free to reopen the issue again if the problems still exist after I fixed #15.
